### PR TITLE
[IOTDB-2963]metrics of process and hardware

### PR DESCRIPTION
**Pull request by ljn55966005**

### CODE REVIEW

**Review by wangchao316 — COMMENTED** [review]
hi, thanks for your contribution.
you could raise a jira, and bind this pr.

File: docs/zh/UserGuide/Maintenance-Tools/Metric-Tool.md [hunk 1]
```diff
@@ -110,25 +110,41 @@ IoTDB对外提供JMX和Prometheus格式的监控指标，对于JMX，可以通
...
 | data_read               | name="compaction"                                                       | important          | 合并文件时的读取量                 | data_read={name="compaction",} 10240                 |
 
-#### 4.3.5. 内存占用
+#### 4.3.5. CPU
+| Metric | Tag                                     | level | 说明                                              | 示例                              |
+| ------ | --------------------------------------- | ------ | -------------------------------------------------- | --------------------------------- |
+| process_cpu_load | name="process" | important | process当前CPU占用率（%） | process_cpu_load{name="process",} 5.0 |
+| process_cpu_time | name="process" | important | process累计占用CPU时间（ns) | process_cpu_time{name="process",} 3.265625E9 |
+| sys_cpu_load | name="system" | important | system当前CPU占用率（%） | sys_cpu_load{name="system",} 15.0 |
+| sys_cpu_cores | name="system"  | important | jvm可用处理器数 | sys_cpu_cores{name="system",} 16.0 |
```
Comment (SpriCoder) [inline docs/zh/UserGuide/Maintenance-Tools/Metric-Tool.md:119]: I think these metrics should be `CORE` level.

File: docs/zh/UserGuide/Maintenance-Tools/Metric-Tool.md [hunk 2]
```diff
@@ -110,25 +110,41 @@ IoTDB对外提供JMX和Prometheus格式的监控指标，对于JMX，可以通
...
+
+#### 4.3.6. 内存占用
 
-| Metric | Tag                                     | 说明   | level                                              | 示例                              |
+| Metric | Tag                                     | level | 说明                                              | 示例                              |
 | ------ | --------------------------------------- | ------ | -------------------------------------------------- | --------------------------------- |
 | mem    | name="chunkMetaData/storageGroup/mtree" | important | chunkMetaData/storageGroup/mtree占用的内存（byte） | mem{name="chunkMetaData",} 2050.0 |
+| process_max_mem | name="process" | important | process最大可用内存 | process_max_mem{name="process",} 3.545759744E9 |
+| process_total_mem | name="process" | important | process当前占用内存 | process_total_mem{name="process",} 2.39599616E8 |
+| process_free_mem | name="process" | important | process当前剩余可用内存 | process_free_mem{name="process",} 1.94035584E8 |
```
Comment (SpriCoder) [inline docs/zh/UserGuide/Maintenance-Tools/Metric-Tool.md:128]: I think these metrics should be `CORE` level.

**Review by SpriCoder — COMMENTED** [review]
Excellent Job! There are only some minor problems need to fix.
Besides, I think some of these metrics should be `CORE` level, please have a check!
Finally, I think you need to give a performance test after introducing these metrics, so have a try of iotdb-benchmark.

**Review by SpriCoder — APPROVED** [review]
LGTM！

**Review by SteveYurongSu — CHANGES_REQUESTED** [review]
Thanks for your wonderful job :D

BTW, Please merge the upstream master into your branch, the CI fails because of the legacy code on upstream master.

**Review by SteveYurongSu — APPROVED** [review]
LGTM

**Pull request merged**